OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/opatch_patch/patches.download

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# Oracle Unified Directory
+# Download patches from : https://support.oracle.com
+#
+# To build the OUD image with OPath patch, you must download and drop the OPatch patch zip file (for e.g. p28186730_139421_Generic.zip) into this directory.
```

Comment:
Typo: `OPatch` but is this file even necessary? Usually the `download` files contain the `md5sum` of the required file to verify the download, but this is just a placeholder. I would remove the file as you've already documented this.

Reply:
In future, we are going to have this file updated with list of patch file names and checksum. Considering usage in future, we would prefer to keep this file.

Reply:
In that case, you can add the file when you have actual patch names and checksums. Having a blank file is just confusing.

Reply:
Currently, README file(s) are not having anything specifically documented about patches. Until we have the README updated with patches related instructions, it would be good if you can let us have this patches.download file where its mentioned to keep downloaded patches in the directory.

Reply:
If the README has no documentation on patches then these files are definitely not necessary either. Until you actually include patch documentation with actual patch names and checksums, all this stuff should be removed to reduce confusion.

Reply:
As our initial plan was to publish only imagetool based approach for building images, details about building images with patches through Dockerfile are not included in the README. Dockerfiles are having logic to pick up patches from those directories and include them in the image. Without details like "To build the OUD image with patches, you must download and drop the OUD 12.2.1.4.0 patch zip files (for e.g. p30851280_122140_Generic.zip) into the patches directory." in patches.download file, users would not be able to determine usage of those folders.

Reply:
Are you finding that the file name `patches.download` is creating confusion, we can rename that to name like `information.txt`. If you can let us, I would prefer to keep some information in those folders until we have the README files updated with details.

Reply:
I think all the folders and files to do with patches should be removed completely _until you actually have patches to apply_. If there are no patches, there is no need for any of these folders. Just get rid of all of them.

Reply:
It would be good if you can let us merge with this file. Otherwise, with removal of patches folder, we would have to make changes in Dockerfile and that will have to go through validation cycle which will delay our merge. We had followed approach about patches.download based on OracleSOASuite (https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/12.2.1.4/patches/soapatches.download)

Reply:
Yeah, the SOA file shouldn't have been merged either. I will compromise: document the directory's purpose in the README.md and use a `.gitkeep` file in the directory to ensure it gets created, but remains empty.